### PR TITLE
Weightless

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # Re-adds an event from old hippie where the grav gen temporarily turns up. I think I fucked up this branch, so just testing. Added literally one file to it and pushed, why is all this other dumb shit showing up like butterfly knives?

:cl: Spankmaster
add: Gravity Disruption event
/:cl:

[why]: # Fun little event pulled from oldschool hippie code

### CODE REVIEW

**Comment by ghost** [comment]
Well this is ugly. I guess I "dirtied" my master. Its 4 am so ill fix this later..

**Comment by ghost** [comment]
“Grav gen temporarily turns up” lol, guess I was super delirious last night.

Anyways I need to reclone my repo looks like. The code in this PR is fully functional and the added files are the same as my hedgehog one. All the event does when triggered is disable the grav gen for about 60 seconds then turn it back on.

Only issue is the way it’s written also effects the derelict grav gen. Not sure how to target the one on station only.

**Comment by ghost** [comment]
Actually no the hedgehog code is fucked up, can’t merge it like that. Does anyone know how I can remove all the files in this PR except weightless.dm? 

I did something with my master that caused it to be clogged with this extra shit but I’m not sure how to fix it

File: hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#define FEAR_DURATION 100
+
+/mob/living/simple_animal/pet/hedgehog
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm:3]: ???

Comment (ghost) [reply hippiestation/code/modules/mob/living/simple_animal/friendly/hedgehog.dm:3]: Yea my master branch is somehow leaking into this PR, it didnt show it until I PR'd. It should just be "weightless" branch with this event code. The hedgehog.dm is identical to what's in my other PR so merging shouldn't be problematic. I'm not sure if the animal.dmi is different.

You got any idea how to clean this shit up? The ONLY thing in the Branch I PR'd was the weightless.dm

**Comment by ghost** [comment]
new PR up for this, closing.

**Pull request closed without merging**